src/themes/default/elements/icon.overrides

```diff
@@ -1322,14 +1360,7 @@ i.icon.write:before { content: "\f303"; }
   i.icon.window.minimize.outline:before { content: "\f2d1"; }
   i.icon.window.restore.outline:before { content: "\f2d2"; }
 
-  /* Outline Aliases */
```

Comment:
Are these not needed anymore?

Reply:
No, I need to manually add them after the script and forgot to.